en/lessons/basics/strings.md

```diff
@@ -37,6 +37,14 @@ iex(6)> "hełło" <> <<0>>
 
 `322` is the Unicode codepoint for ł but it is encoded in UTF-8 as the two bytes `197`, `130`.
 
+You can get a character’s code point by using ?
+
+```elixir
+iex(8)> ?Z  
```

Comment:
Please remove the `(8)`, we don't include the `iex` line number (you can disable it btw) in other lessons.

Reply:
@doomspork would you consider moving our styleguide into `CONTRIBUTING.md`? This is referenced in there and it might make these nits come through less often if we consolidated them into one thing. Its in the style guide, and contributing includes adhering to the style guide per its own words, so it may be worth it to save a click and just make it part of the baseline there.

Though I have no strong feelings on this, just have noticed this cuz its a mistake I used to make too haha (I generally check `CONTRIBUTING.md` before opening a PR on a project I dont commit to weekly or more just to be sure I'm following the latest guidelines, though this is obviously just my usecase and not everyones.)

Reply:
We include a link. I met including the entirety of it as a subsection at the bottom.